theory-test-start-admission/lambda.js

```diff
@@ -1,5 +1,34 @@
+require('dotenv').config();
+
+const AdmissionDAO = require('theory-test-admission-dao');
 const logger = require('logger');
-const main = require('./src/main');
+
+exports.handler = (event, context, callback) => {
+
+	// log inbound event
+	logger.debug('Received event: ', JSON.stringify(event));
+	logger.info('Running start admission');
+	let response;
+	if (event.DrivingLicenceNumber && event.AdmissionId) {
+		const parameters = {
+			DrivingLicenceNumber: event.DrivingLicenceNumber,
+			HasBooking: event.HasBooking,
+			AdmissionId: event.AdmissionId
+		};
+
+		const admission = AdmissionDAO.createAdmissionDatabaseRecord(parameters);
+		AdmissionDAO.create(admission, (err, retVal) => {
+			if (!err) {
```

Comment:
...and if there _is_ an error?

Reply:
else {
			logger.error('The following error occurred when trying to create a new admission record: ', err);
			exit(callback, err, null);
		}

Reply:
Should I return null as the response, or should i generate a failed response?